internal/controller/controller.go

```diff
@@ -234,7 +239,7 @@ func (c *httpController) exportRecordedData(writer http.ResponseWriter, request
 
 	if reflect.DeepEqual(recordedData, &dtos.RecordedData{}) {
 		writer.WriteHeader(http.StatusNoContent)
-		_, _ = writer.Write([]byte("no recorded data found"))
+		_, _ = writer.Write([]byte(noDataFound))
 		return
 	}
```

Comment:
`ExportRecordedData` should return an error when there isn't any recorded data, so this check should be removed.